2check/app/enums/panicEffects.ts

```diff
@@ -0,0 +1,39 @@
+export enum PanicEffects {
```

Comment:
good job here

Reply:
also you can think about moving all effects into classes, so you will have class for each effect, and inside this class you can encapsulate calculations related to this effect.